### PR TITLE
Allow to work with phpBB 3.2

### PR DESCRIPTION
**Pull request by VSEphpbb**
This should be ready for review
@phpbb/development-team @phpbb/extensions-team 

It seems to be working now, but I bet some of these changes could have been done a little better ;)

### CODE REVIEW

**Comment by VSEphpbb** [comment]
Bump
@phpbb/development-team @phpbb/extensions-team

File: modules/qi_create.php [hunk 1]
```diff
@@ -555,10 +576,127 @@ public function __construct()
...
+
+			$iohandler->set_input('cookie_secure', $settings->get_config('cookie_secure', 0));
+			$iohandler->set_input('server_protocol', $settings->get_server_protocol());
+			$iohandler->set_input('force_server_vars', 'http://');
+			$iohandler->set_input('server_name', $settings->get_config('server_name'));
+			$iohandler->set_input('server_port', $settings->get_config('server_port'));
+			$iohandler->set_input('script_path', $script_path);
+			$iohandler->set_input('submit_server', 'submit');
+
+			//$installer->run();
```
Comment (Nicofuma) [inline modules/qi_create.php]: why is this line commented?

Comment (VSEphpbb) [inline modules/qi_create.php]: Because it's shouldn't be used, but I didn't want to erase it yet...I think

**Comment by DavidIQ** [comment]
I'll properly review/test this in the next few days.

**Comment by DavidIQ** [comment]
I don't really like that this is keeping compatibility with 3.0, which is EOL already, but not sure if that should be another ticket.  I'd be ok with it if it wasn't.

Otherwise the code is good.  I'll try to test this out over the weekend.

**Comment by Un1matr1x** [comment]
I would prefer to stay with 3.0 atm - it doesn't hurt anybody an mod-devs could still support their mods

you could mark it as abdoned in the ui

**Comment by VSEphpbb** [comment]
3.0 is still necessary. For example, for ext authors that want to port over a MOD. And of course MOD authors still have MODs in use out there and may still have support issues to deal with from users. These are just two cases where it'd still be good to have a way to quick install a 3.0 board.

I think, as long as 3.0.x is still downloadable from dot com, we should support it here, even though it's support at this point here is simply a legacy.

**Comment by DavidIQ** [comment]
Works fine as long as you're not on PHP 7.  (see #55)
Once that gets resolved 1.2.2 should be tagged for release in the tools section of phpbb.com

**Pull request merged**